spec/support/acts_as_tenant.rb

```diff
@@ -19,9 +19,15 @@ def with_tenant(tenant, &proc)
       context "with tenant #{tenant.inspect}" do |*params|
         before(:each) do
           ActsAsTenant.current_tenant = send(tenant)
+
+          # it's recommended to use test_tenant as it's thread-safe for
+          # testing in parallel settings, which we do.
+          # Please refer to https://github.com/ErwinM/acts_as_tenant/issues/277
+          ActsAsTenant.test_tenant = send(tenant)
```

Comment:
`send(tenant)` called twice, isn't it better to store it as a variable then use that variable twice instead?

Reply:
done